src/wp-includes/block-supports/background.php

```diff
@@ -40,6 +40,8 @@ function wp_register_background_support( $block_type ) {
  * it is also applied to non-server-rendered blocks.
  *
  * @since 6.4.0
+ * @since 6.5.0 Added support for `backgroundSize`, `backgroundPosition`,
```

Comment:
`backgroundSize` was added in 6.4?

Reply:
Good catch! Updated in https://github.com/WordPress/wordpress-develop/pull/5828/commits/6a82e04599d0e5b9a486335787136c77af3c1dd4. The output of `backgroundSize` was added in 6.4 but the UI controls weren't. Since this function already handled output for `backgroundSize`, I've updated the comment here to just reflect position and repeat output.